cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -0,0 +1,529 @@
+/*
+ * Copyright 2017-2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model;
+
+import io.fabric8.kubernetes.api.model.Container;
+import io.fabric8.kubernetes.api.model.ContainerBuilder;
+import io.fabric8.kubernetes.api.model.ContainerPort;
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.IntOrString;
+import io.fabric8.kubernetes.api.model.LocalObjectReference;
+import io.fabric8.kubernetes.api.model.Service;
+import io.fabric8.kubernetes.api.model.ServicePort;
+import io.fabric8.kubernetes.api.model.Volume;
+import io.fabric8.kubernetes.api.model.VolumeMount;
+import io.fabric8.kubernetes.api.model.apps.Deployment;
+import io.fabric8.kubernetes.api.model.apps.DeploymentStrategy;
+import io.fabric8.kubernetes.api.model.apps.DeploymentStrategyBuilder;
+import io.fabric8.kubernetes.api.model.apps.RollingUpdateDeploymentBuilder;
+import io.fabric8.kubernetes.api.model.policy.PodDisruptionBudget;
+import io.strimzi.api.kafka.model.CertAndKeySecretSource;
+import io.strimzi.api.kafka.model.CertSecretSource;
+import io.strimzi.api.kafka.model.KafkaBridgeConsumerSpec;
+import io.strimzi.api.kafka.model.KafkaBridgeHttpConfig;
+import io.strimzi.api.kafka.model.KafkaBridge;
+import io.strimzi.api.kafka.model.KafkaBridgeAuthenticationPlain;
+import io.strimzi.api.kafka.model.KafkaBridgeAuthenticationScramSha512;
+import io.strimzi.api.kafka.model.KafkaBridgeAuthenticationTls;
+import io.strimzi.api.kafka.model.KafkaBridgeProducerSpec;
+import io.strimzi.api.kafka.model.KafkaBridgeSpec;
+import io.strimzi.api.kafka.model.KafkaBridgeTls;
+import io.strimzi.api.kafka.model.PasswordSecretSource;
+import io.strimzi.api.kafka.model.Probe;
+import io.strimzi.api.kafka.model.ProbeBuilder;
+import io.strimzi.api.kafka.model.template.KafkaBridgeTemplate;
+import io.strimzi.operator.common.model.Labels;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class KafkaBridgeCluster extends AbstractModel {
+
+    // Port configuration
+    protected static final int DEFAULT_REST_API_PORT = 8080;
+    protected static final String REST_API_PORT_NAME = "rest-api";
+
+    protected static final int HEALTH_CHECK_PORT = 8081;
+    protected static final String HEALTH_CHECK_PORT_NAME = "healthcheck";
+
+    private static final String NAME_SUFFIX = "-bridge";
+    private static final String SERVICE_NAME_SUFFIX = NAME_SUFFIX + "-api";
+
+    private static final String METRICS_AND_LOG_CONFIG_SUFFIX = NAME_SUFFIX + "-config";
+    protected static final String TLS_CERTS_BASE_VOLUME_MOUNT = "/opt/bridge/bridge-certs/";
+    protected static final String PASSWORD_VOLUME_MOUNT = "/opt/bridge/bridge-password/";
```

Comment:
I think this should be `/opt/strimzi` since that is there we install the bridge.

```suggestion
    protected static final String PASSWORD_VOLUME_MOUNT = "/opt/strimzi/bridge-password/";
```